package.json

```
@@ -103,6 +103,7 @@
     "@playwright/experimental-ct-react": "^1.36.2",
     "@playwright/test": "^1.36.2",
     "@storybook/addon-essentials": "7.2.1",
+    "@storybook/addon-storysource": "^7.2.2",
```

```suggestion
    "@storybook/addon-storysource": "7.2.1",
```

Please revert the changes and redo with a fixed version (same for all storybook packages). Storybook doesn't like it when we mix up package versions.